packages/server/src/users/api/UserDto.ts

```diff
@@ -0,0 +1,53 @@
+import { ApiProperty, ApiPropertyOptional, OmitType } from '@nestjs/swagger';
+
+import { FavouriteRestaurantDto } from '../../restaurants/api/RestaurantDto';
+import { FavouriteDishDto } from '../../restaurants/dishes/api/DishDto';
+import { ApiObjectIdProperty } from '../../shared';
+
+interface Card {
+  number: number;
+  expirationDate: Date;
+  securityCode: number;
```

Comment:
To co wyżej, są kody 3 i 4 cyfrowe, zapisując w bazie 123 nie wiemy potem czy to było `123` czy `0123`.